rust/parser/test/mod.rs

```diff
@@ -167,10 +167,10 @@ fn when_parsing_query_with_comments_they_are_ignored() {
     let query = r#"match
 # there's a comment here
 $x isa###WOW HERES ANOTHER###
-movie; count();"#;
+movie; count($x);"#;
```

Comment:
The equivalent of `count()` before is `count` without parentheses. This is not any more or less correct though.